calendar.js

```diff
@@ -24,17 +24,25 @@ EntrepreneurshipFeed.setDomain('techatnyu.org').setName('Tech@NYU Entrepreneursh
 
 // Prep teams array
 var teamIdsToRoleNames = {};
+var venueIdsToVenueAddresses = {};
 request({url: 'https://api.tnyu.org/v2/teams?include=memberships', rejectUnauthorized: false})
     .then(function(teamsBody) {
         var teams = JSON.parse(teamsBody).data;
         for (var i = 0; i < teams.length; i++) {
             var currentTeam = teams[i];
             teamIdsToRoleNames[currentTeam.id] = currentTeam.roleName;
-        };
+        }
 
-        return request({url: 'https://api.tnyu.org/v2/events', rejectUnauthorized: false});
+        return request({url: 'https://api.tnyu.org/v2/events?include=venue', rejectUnauthorized: false});
     }).then(function(eventsBody) {
         var events = JSON.parse(eventsBody).data;
+        var venues = JSON.parse(eventsBody).included;
```

Comment:
Minor performance win: you could save the `JSON.parse(eventsBody)` to a variable, so that you don't have to parse it twice. Not a huge deal though.